evmcore/tx_pool.go

```diff
@@ -555,6 +555,17 @@ func (pool *TxPool) Pending(enforceTips bool) (map[common.Address]types.Transact
 	return pending, nil
 }
 
+func (pool *TxPool) PendingSlice() types.Transactions {
+	pool.mu.Lock()
+	defer pool.mu.Unlock()
+
+	pending := make(types.Transactions, 0, 1000)
```

Comment:
are you sure, that 1000 is the sufficient upper bound for pending items?

Reply:
@cyberbono3 I think that it's only the initialize capacity, slice will grow if it need more when using `append` func